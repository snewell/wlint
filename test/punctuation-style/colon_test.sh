#!/bin/sh

current_dir=$(dirname ${0})
current_script=$(basename ${0})

input="${current_dir}/${current_script}.in"
wlint punctuation-style ${input} |
	sed "s^${input}^<stdin>^g" |
	diff -u "${current_dir}/${current_script}.out" -
