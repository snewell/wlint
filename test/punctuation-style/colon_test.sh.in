Skittles come in these colors : red, green, orange, yellow, and purple.

The fire swamp has three dangers :lightning sand, fire spurts, and rodents of unusual size.

I once knew a man from Nantucket ;he wasn't that interesting.

The dog napped all day ; not even food could wake her.
