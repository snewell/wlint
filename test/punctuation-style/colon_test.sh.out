<stdin>-1:29 colon.preceeding-space
<stdin>-3:32 colon.preceeding-space
<stdin>-3:33 colon.missing-space
<stdin>-5:32 semicolon.preceeding-space
<stdin>-5:33 semicolon.missing-space
<stdin>-7:22 semicolon.preceeding-space
